src/catalog/product-create-bundle.md

```diff
@@ -136,6 +136,9 @@ _Choose Template_
     ![]({% link images/images/product-bundle-items.png %}){: .zoom}
     _Bundle Items_
 
+    If you select `Together` and have enabled _In-Store Delivery_, all bundle items must be assigned the same [source]({% link catalog/inventory-sources.md %}).
```

Comment:
In-store pickup does not affect this statement. To ship bundle products with option "Together" all items must be assigned to the same source.